### PR TITLE
Add Mock Service Worker icon

### PR DESCRIPTION
**Pull request by arcyopterix**
**Issue:** Add Mock Service Worker (MSW) icon
**Similarweb rank:** [893,428](https://www.similarweb.com/website/mswjs.io/#overview)
**Github stars:** [12.8k](https://github.com/mswjs/msw)

### Checklist
  - [X] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### Description
Adding MSW icon as requested in issue #6160

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5751,6 +5751,11 @@
             "hex": "8D6748",
             "source": "https://mochajs.org/"
         },
+        {
+            "title": "Mock Service Worker",
+            "hex": "FF6A33",
+            "source": "https://github.com/mswjs/msw/blob/master/media/msw-logo.svg"
```
Comment (ericcornelissen) [inline _data/simple-icons.json:5757]: As per the [Contributing Guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#source-guidelines), this link should include the exact commit hash.
```suggestion
            "source": "https://github.com/mswjs/msw/blob/5e69e91f79af181a544dea290a7fa4391d8d2ad4/media/msw-logo.svg"
```

**Review by ericcornelissen — CHANGES_REQUESTED** [review]
The Mock Service Worker SVG looks spot on, nice work @arcyopterix! I have one mall request for a change related to the metadata for Mock Service Worker though.

Also, as you already opened #6139 for the Skynet icon, could you remove the file and metadata related to that from this Pull Request? You can get fancy and cherrypick the commits in git, but from our side it's fine if you just remove the file and date in a new commit (this is because we sqaush-and-merge, so the commit history is lost anyway).

**Comment by adamrusted** [comment]
After a _lot_ of dabbling, I've gotten this PR up to speed with a newer variation of the logo. @simple-icons/icon-review for reference.

mockserviceworker

**Review by PratyushJayachandran — APPROVED** [review]
Thank you for starting the PR @arcyopterix and thanks @adamrusted for finishing it up. Perfect match 🎉

**Pull request merged**